src/cli/onefuzz/backend.py

```diff
@@ -45,16 +45,24 @@ def _temporary_umask(new_umask: int) -> Generator[None, None, None]:
             os.umask(prev_umask)
 
 
+class BackendConfig(BaseModel):
+    authority: str
+    client_id: str
+    client_secret: Optional[str]
+    endpoint: Optional[str]
+    features: Dict[str, bool] = Field(default_factory=dict)
```

Comment:
Note, this uses str, rather than the enum used by the API.  Eventually, feature flags should be removed.  We don't want to fail in parsing if users have old feature flags added.  Rather, we should just ignore them.